.rubocop_todo.yml

```diff
@@ -89,6 +89,7 @@ RSpec/VerifiedDoubles:
 
 RSpec/AnyInstance:
   Exclude:
+    - 'spec/controllers/api/generic_works_controller_spec.rb'
```

Comment:
Any hints on how to go about fixing it, I wasn't sure why this one was failing but the items_controller_spec and zotero_controller_spec were passing (as this controller was based on).. we found these were being ignored in the rubocop_todo as well. 

Are we missing a simple fix that we could implement into the item_controller_spec and zotero_controller_spec as well?